docs/data/sdks/ios-swift/index.md

```diff
@@ -97,13 +97,25 @@ amplitude.identify(identify: identify)
 
 --8<-- "includes/groups-intro-paragraph.md"
 
-For example, if Joe is in 'orgId' '10' and '16', then the `groupName` would be '[10, 16]'). Here is what your code might look like:
```

Comment:
Why remove this description? I think we should call out that groupName can be either a single name or an array for all platforms.

Reply:
+1

Reply:
Moved it because there is description above in "includes/groups-intro-paragraph.md" and it is not aligned with the code. 

Fixed the description  and put it to an example admonition